docs/reference/tools-reference/llm-tool.md

```diff
@@ -158,3 +158,69 @@ In LLM tool, the prompt is transformed to match the [openai messages](https://pl
     }
 ]
 ```
+
+### Sample 3
+This sample illustrates how to write a tool chat prompt.
+```jinja
+# system:
+You are a helpful assistant.
+
+# user:
+What is the current weather like in Boston?
+
+# assistant:
+{# The assistant message with 'tool_calls' must be followed by a tool message. #}
+## tool_calls:
+{{outputs.llm_output.tool_calls}}
+ 
+{# The tool message must follow an assistant message with 'tool_calls'. #}
```

Comment:
messages with role 'tool' must be a response to a preceeding message with 'tool_calls'